src/Http/Controllers/ScriptController.php

```diff
@@ -7,9 +7,19 @@
 class ScriptController
 {
     public function __invoke(Request $request)
+    {
+
+    }
+
+    public function getCookieScript()
     {
         $content = str_replace('{config:1}', $this->generateConfig(), file_get_contents(LCC_ROOT . '/dist/cookies.js'));
+        return response($content)->header('Content-Type', 'application/javascript');
+    }
 
+    public function getModalScript()
+    {
+        $content = file_get_contents(LCC_ROOT . '/dist/modal.js');
         return response($content)->header('Content-Type', 'application/javascript');
     }
```

Comment:
Maybe use this opportunity to pass the translation strings to the modal's constructor, like we did for the API routes configuration in `getCookiesScript()` ?